lib/process_settings/settings.rb

```diff
@@ -11,6 +11,8 @@ class Settings
     def initialize(json_doc)
       json_doc.is_a?(Hash) or raise ArgumentError, "ProcessSettings must be a Hash; got #{json_doc.inspect}"
 
+      AbstractMonitor.ensure_no_symbols(json_doc)
```

Comment:
I saw some symbols go live in invoca_process_settings... so I wanted to add an explicit contract to reject those.